internal/services/machinelearning/machine_learning_workspace_resource.go

```diff
@@ -123,6 +123,19 @@ func resourceMachineLearningWorkspace() *pluginsdk.Resource {
 				DiffSuppressFunc: suppress.CaseDifference,
 			},
 
+			"public_network_access": {
```

Comment:
```suggestion
			"public_network_access_enabled": {
```

Reply:
This looks okay

Reply:
How has this field been called for other RP's @katbyte @xuzhang3. Via ARM and API this field is called "public_network_access" and of type enum with "Enabled" / "Disabled" per compliance requirement.

Reply:
recent resources are all using `public_network_access_enabled` as a boolean.

Reply:
Then we can follow that pattern as well